codegen/src/main/java/io/spine/tools/java/Directory.java

```diff
@@ -31,27 +32,46 @@
  *
  * @author Alexander Yevsyukov
  */
-public final class Folder extends AbstractFolder<FileName, SourceFile> {
+public final class Directory extends SourceCodeDirectory {
```

Comment:
There are two classes named `Directory` each under the language package: `io.spine.tools.java` and `io.spine.tools.proto`.

The intent was to have a shorter directory class name and have language processing more-or-less isolated. There are a couple of places where we have to use FQNs because of that, but in the rest the code looks more compact.

I suggest to return to this matter when we have more code that depends on the new code processing API. It may turn that these classes would be transformed to something else, when we rely on `Descriptor`s (rather than `DescriptorProto`s) API.